specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+          }
+        }
+      }
+    },
+    "/execution/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/experiments/{experimentName}/runId/{runId}/cancel": {
+      "post": {
+        "tags": [
+          "Execution"
+        ],
+        "summary": "Cancel a run.",
+        "description": "Cancels a run within an experiment.",
+        "operationId": "Execution_CancelRunWithUri",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "experimentName",
+            "in": "path",
+            "description": "The experiment name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "runId",
+            "in": "path",
+            "description": "The id of the run to cancel.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9][\\w-]{0,255}$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The run was successfully cancelled.",
+            "schema": {
+              "$ref": "#/definitions/RunStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Cancel a run with URI": {
+            "$ref": "./examples/cancelRunWithUri.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RunStatus": {
+      "description": "Contains the details of a run.",
+      "type": "object",
+      "properties": {
+        "runId": {
+          "description": "The identifier for a run.",
+          "type": "string",
+          "example": "myexperiment_155000000001_0"
+        },
+        "target": {
+          "description": "Target refers to compute where the job is scheduled for execution.",
+          "type": "string",
+          "example": "local"
+        }
+      }
+    },
+    "RunDefinition": {
+      "type": "object",
+      "properties": {
+        "configuration": {
+          "$ref": "#/definitions/RunConfiguration",
+          "description": "Fully specified configuration information for the run. Even when that information\r\nis contained in configuration files within the project folder, the client collapses\r\nit all and inlines it into the run definition when submitting a run."
+        },
+        "attribution": {
+          "type": "string"
+        },
+        "telemetryValues": {
```

Comment:
Can you give some examples on what kind of objects you would expect to match this schema definition? I suspect that it does not correctly describe the format that your service accepts. E.g. it would not allow for something like:

```json
"telemetryValues": {
    "something": "else"
}
```

since the value of the `something` property is not an instance of an `object`.

Reply:
This field should not be documented for third-parties.

Reply:
done